zstacklib/zstacklib/utils/linux.py

```
@@ -3365,3 +3365,77 @@ def catch_bad_alloc_exception(return_code, error_detail):
+DirectMap2M:     5103616 kB
+DirectMap1G:    13631488 kB
+    """
+    try:
+        # 优先使用 MemAvailable
+        with open('/proc/meminfo') as f:
+            for line in f:
+                if line.startswith('MemAvailable:'):
+                    parts = line.split()
+                    return int(parts[1]) * 1024  # kB -> bytes
+
+        # 回退：MemFree + Buffers + Cached
```

Ruff (0.13.1)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)